assets/product-info.js

```diff
@@ -59,7 +59,7 @@ if (!customElements.get('product-info')) {
 
       fetchQuantityRules() {
         if (!this.currentVariant || !this.currentVariant.value) return;
-        this.querySelector('.quantity__rules-cart .loading-overlay').classList.remove('hidden');
+        this.querySelectorAll('.quantity__rules-cart .loading-overlay').forEach((overlay) => overlay.classList.remove('hidden'));
```

Comment:
Showing loading spinner could also be handled by the new `price-per-item` component instead